ServantPersistent/src/Price.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving  #-}
+{-# LANGUAGE DeriveGeneric  #-}
+module Price where
+
+import Database.Persist.Sql
+import GHC.Generics
+import Data.Aeson
+
+newtype Price = Price { intPrice :: Int }
```

Comment:
This type should map to `NUMERIC` in the DB, which has the ability to represent fractionals without loss of precision (basically, it's not a standard float).